samples/system-test/inspect.test.js

```diff
@@ -50,10 +52,9 @@ test.after.always(async () => {
   await subscription.delete().then(() => topic.delete());
 });
 
-// inspect_string
-test(`should inspect a string`, async t => {
+test(`dlp_inspect_string`, async t => {
   const output = await tools.runAsync(
-    `${cmd} string "I'm Gary and my email is gary@example.com"`,
+    `node dlp_inspect_string.js ${PROJECT_ID} "I'm Gary and my email is gary@example.com"`,
```

Comment:
This means you can never run with another `node` except the one that's in the path. Not a huge fan.

Reply:
All the sample tests in this repo have that problem.